### PR TITLE
NEW PIANO SAMPLES!

### PR DESCRIPTION
**Pull request by ithebinman**
## About the PR 

I replaced the old samples of the player piano and added new crisp ones, as well as extra octaves!
**The full range now is from Cb1 to Bs6!**

I created new samples for the Player Piano keys using  free creative commons piano sound files.

for more information about the original samples and their licensing under creative commons see:

Selections from The Berklee College of Music Sampling Archive (1999-2007). Found on: https://wiki.laptop.org/go/Free_sound_samples
## Why's this needed? 

Many songs are limited with the current range of the player piano, as well as by the low quality timbre of the current samples, so I upgraded them!

### CODE REVIEW

**Comment by nefarious6th** [comment]
yay! 🎊

**Comment by ithebinman** [comment]
whoops I pressed the "ready for review button" by mistake THIS IS STILL A DRAFT!!!

**Comment by DimWhat** [comment]
Reopening upon request of the author

**Comment by ithebinman** [comment]
aight so for anyone interested:
1: turns out this doesn't even need an edit to the code, as Todd Howard once said, "it just works" so this could be implemented at any time.
2: after testing the sounds in game I'm not happy with how they sound, PLUS I am going to try and add a 7th octave cause people were asking for a full piano roll so that's fair.

TLDR: this PR works, but I wanna work on the sounds to be higher quality
